clients/drcachesim/tracer/output.cpp

```diff
@@ -1005,13 +1008,30 @@ process_and_output_buffer(void *drcontext, bool skip_size_cap)
 
     if (do_write) {
         bool hit_window_end = false;
-        if (op_trace_for_instrs.get_value() > 0) {
+
+        if (op_L0_filter_until_instrs.get_value() && mode == BBDUP_MODE_L0_FILTER) {
+            uintptr_t toadd =
+                *(uintptr_t *)TLS_SLOT(data->seg_base, MEMTRACE_TLS_OFFS_ICOUNT);
+            hit_window_end = count_traced_instrs(drcontext, toadd,
+                                                 op_L0_filter_until_instrs.get_value());
+            if (hit_window_end) {
+                data->bytes_written = 0;
+                NOTIFY(0, "Adding endpoint marker\n");
```

Comment:
s/endpoint marker/filter endpoint marker for -L0_filter_until_instrs/

Reply:
Done.